src/evalfunc.c

```diff
@@ -5216,6 +5211,47 @@ f_get(typval_T *argvars, typval_T *rettv)
 	copy_tv(tv, rettv);
 }
 
+/*
+ * "getcellpixels()" function
+ */
+    static void
+f_getcellpixels(typval_T *argvars UNUSED, typval_T *rettv)
+{
+    if (rettv_list_alloc(rettv) == FAIL)
+        return;
+
+#if defined(FEAT_GUI)
+    if (gui.in_use)
+    {
+        // success pixel size and no gui.
+        list_append_number(rettv->vval.v_list, (varnumber_T)gui.char_width);
+        list_append_number(rettv->vval.v_list, (varnumber_T)gui.char_height);
+    }
+    else
+    {
+#endif
+        struct cellsize cs;
+# if defined(UNIX)
+        mch_calc_cell_size(&cs);
+# else
```

Comment:
```suggestion
#else
```

Reply:
Fixed in 7818a497285a0c416703698c4c20cc3006acbe75.